charts/datadog/values.yaml

```diff
@@ -233,6 +233,12 @@ datadog:
   #   kubernetes.io/role: kube_role
   #   <KUBERNETES_NODE_LABEL>: <DATADOG_TAG_KEY>
 
+  # datadog.nodeAnnotationsAsTags -- Provide a mapping of Kubernetes Node Annotations to Datadog Tags
+  nodeAnnotationsAsTags:
+    cluster.k8s.io/machine: kube_machine
```

Comment:
This should be commented

Reply:
I intend to have this as a default value

Reply:
see [here](https://github.com/DataDog/datadog-agent/blob/6c570633d6cc089c433d55bf4b2fba7e83d9943a/pkg/config/setup/config.go#L355)

Reply:
It might be better to not set any default here and let the one in the agent apply if there's no value specified here.